Ollivanders/src/net/pottercraft/Ollivanders2/Spell/APARECIUM.java

```diff
@@ -44,10 +44,9 @@ public APARECIUM (Ollivanders2 plugin, Player player, Spells name, Double rightW
    public void checkEffect ()
    {
       move();
-      List<StationarySpellObj> stationaries = p.checkForStationary(location);
+      List<StationarySpellObj> stationaries = p.stationarySpells.getStationarySpellsAtLocation(location);
```

Comment:
We should redo that invisibility check to use this function so instead of checking all the stationary spells we just check the ones at the persons location.

Reply:
https://github.com/Azami7/Ollivanders2/issues/144 filed to update invis check.